sdk/metric/controller/push/config.go

```diff
@@ -73,3 +76,13 @@ type timeoutOption time.Duration
 func (o timeoutOption) Apply(config *Config) {
 	config.Timeout = time.Duration(o)
 }
+
+func WithMetricsProcessor(p metric.MetricsProcessor) Option {
```

Comment:
```suggestion
func WithMetricsProcessors(p ...metric.MetricsProcessor) Option {
```

Perhaps make this variadic as well to match the other option's interface.